docs/cli.md

```diff
@@ -197,6 +197,56 @@ Usage:
       --validate=false: If true, use a schema to validate the input before sending it
 ```
 
+#### resize
+Set a new size for a resizable resource (currently only Replication Controllers)
+
+Resize also allows users to specify one or more _preconditions_ for the resize action.  The two currently supported options are current size (```--current-size```) or resource version (```--resource-version```).  If a precondition is specified, it is validated before the resize is attempted, and it is guaranteed that the precondition holds true when the resize is sent to the server.
+
+Examples:
+```sh
+  $ kubectl resize controllers foo 3
+  resized
+```
+
+```sh
+  # will only execute if the current size is 3
+  $ kubectl resize --current-size=2 controllers foo 3
+```
+
+Usage:
+```
+  kubectl resize [-r/--resource-version] [-s/--current-size] <resource> <id> <count> [flags]
```

Comment:
There are a bunch of PRs in flight (e.g., #3198, #3196, #3004, #2751) that add support for various ways of specifying multiple objects on the command line: files, directories, URLs, label selectors, multiple object names (#3050), ... While multiple objects may make less sense for resize, could we still adhere to the same pattern, or at least a subset? I think that implies that the count should be a flag. Also, for consistency, we might as well use `<size>` rather than `<count>`.

Don't flags have to precede non-flag arguments?